mm2src/coins/utxo/wallet_connect.rs

```diff
@@ -141,3 +147,335 @@ pub async fn get_pubkey_via_wallatconnect_signature(
 
     Ok(pubkey.to_string())
 }
+
+/// The response from WalletConnect for `signPsbt` request.
+#[derive(Deserialize)]
+struct SignedPsbt {
+    psbt: String,
```

Comment:
nit: here and in SignMessageResponse struct we could use `#[serde(with="base64_serde")]` and add a mod to impl ser/deser with base64:
```
mod base64_serde {
    use serde::{Serialize, Deserialize};
    use serde::{Deserializer, Serializer};
    use base64::Engine;

    pub fn serialize<S: Serializer>(v: &Vec<u8>, s: S) -> Result<S::Ok, S::Error> {
        let base64 = super::BASE64_ENGINE.encode(v);
        String::serialize(&base64, s)
    }
    
    pub fn deserialize<'de, D: Deserializer<'de>>(d: D) -> Result<Vec<u8>, D::Error> {
        let base64 = String::deserialize(d)?;
        super::BASE64_ENGINE.decode(base64.as_bytes())
            .map_err(|e| serde::de::Error::custom(e))
    }
}
```

Reply:
done [9df9abd](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2566/commits/9df9abd6e5e66bee1d93025d747a06648ef47842)

im not really sure where this mod fits though. do you have any idea?